modules/cloud_logs/README.md

```diff
@@ -0,0 +1,78 @@
+# Cloud Logs instance sub-module
+
+This sub-module supports provisioning the following observability instances:
+
+- **IBM Cloud Logging with Cloud Logs**
+  - View, analyze, and alert on activity tracking events and logging activity.
+
+:information_source: This sub-module also supports passing COS bucket details to store your IBM Cloud Logs data for long term storage, search, analysis and alerting.
+
+## Usage
+
+To provision Cloud Logs instance
+
+```hcl
+# required ibm provider config
+provider "ibm" {
+  ibmcloud_api_key = var.ibmcloud_api_key #pragma: allowlist secret
+}
+
+# IBM clou logs
+module "cloud_logs" {
+  source  = "terraform-ibm-modules/observability-instances/ibm//modules/cloud_logs"
+  version = "latest" # Replace "latest" with a release version to lock into a specific release
```

Comment:
use X.X.X instead of latest (someone once though latest was a supported string that could be passed)